iterative_verifier.go

```diff
@@ -239,10 +241,37 @@ func (v *IterativeVerifier) VerifyBeforeCutover() error {
 	return err
 }
 
+func (v *IterativeVerifier) reverifyUntilStoreIsSmallEnough(maxIterations int) error {
+	var timeToVerify time.Duration
+
+	for iteration := 0; iteration < maxIterations; iteration++ {
+		before := v.reverifyStore.RowCount
+		start := time.Now()
+
+		_, err := v.verifyStore("reverification_before_cutover", []MetricTag{{"iteration", string(iteration)}})
+		if err != nil {
+			return err
+		}
+
+		after := v.reverifyStore.RowCount
+		timeToVerify = time.Now().Sub(start)
+
+		if after >= before {
```

Comment:
Might it be worth quitting if you're either _above_, or _below_ by e.g. 2-3%? Too complicated?

Reply:
I like this heuristic much more than time, by the way. So much simpler.

Reply:
I'm skeptical about whether the 2-3% breakout is worth the additional math (not that it's too complicated)